Adjust doc about dangling indices after node is detached from cluster (#92824)
elastic/elasticsearch: docs/reference/commands/node-tool.asciidoc

surviving master-eligible node. If there are no remaining master-eligible nodes
then the cluster metadata is completely lost. However, the individual data
nodes also contain a copy of the index metadata corresponding with their
shards. It is therefore sometimes possible to manually import these shards as
<<dangling-indices,dangling indices>>. For example you can sometimes recover some
indices after the loss of all master-eligible nodes in a cluster by creating a new
cluster and then using the `elasticsearch-node detach-cluster` command to move any

> surviving nodes into this new cluster. Once the new cluster is fully formed then
> the <<dangling-indices-api,Dangling indices API>> can be used to list the dangling
> indices that can be imported.

Reviewer: wording nit (we seem to prefer this style):

```suggestion
surviving nodes into this new cluster. Once the new cluster is fully formed,
use the <<dangling-indices-api,Dangling indices API>> to list, import or delete
any dangling indices.
```
